### PR TITLE
better initialize

### PR DESCRIPTION
**Pull request by sofuture**
Initialize with straight cqlsh instead of relying on pithos to setup tables

### CODE REVIEW

**Comment by sofuture** [comment]
marking wip: realized this doesn't take cluster size into account for replication factor.

**Comment by klizhentas** [comment]
@r0mant will need to review

**Comment by r0mant** [comment]
Well, as far as I understand this just creates the same tables pithos would set up itself, and table definitions are taken from cqlsh "describe keyspace" output. I guess my only question would be if there's any benefit of creating them manually instead of letting pithos take care of it? To be able to change something?

**Comment by sofuture** [comment]
There was one install @klizhentas debugged where pithos only half
initialized, we didn't have logs so weren't sure exactly why. This might
not be any better, but is at least more straightforward to debug should
that happen again, and will fail instead of erroneously succeeding.

**Comment by sofuture** [comment]
Okay, sets replication factor again. @r0mant can you give me a :shipit: at some point.

File: images/cassandra/initialize.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+NUM_NODES=$(kubectl get nodes --selector=pithos-role=node --no-headers | wc --lines)
+REPLICATION_FACTOR=1
+
+if [[ $NUM_NODES -gt 2 ]]; then
+	REPLICATION_FACTOR=3
+fi
+
+sed -i 's/REPLICATION_FACTOR/'"${REPLICATION_FACTOR}"'/g' /init.cql
```
Comment (r0mant) [inline images/cassandra/initialize.sh]: Does this work? Don't you need to escape the inner apostrophes?

Comment (sofuture) [inline images/cassandra/initialize.sh]: It does work, I'm trying to not include any of the apostrophes or quotes in the file, just want the value.

**Pull request merged**